content/community/changelog/app-nuget/v4/whats-new/_index.en.md

```diff
@@ -3,6 +3,10 @@ title: What's new
 description: Overview of changes introduced in version 4.
 toc: true
 ---
+## 4.24.0 (2020-01-21)
+
+Support language and query parameters in dynamic options.
+New way of implementing dynamic options by creating a class that implements IAppOptionsProvider.
```

Comment:
Could we link to the documentation explaining how to set this up here in the release notes?

Reply:
Good idea, done:)